Image_Viewer/file_handler.py

```diff
@@ -2,7 +2,7 @@
 from tkinter import filedialog
```

Comment:
Can you remove the `__pycache__` folder please? This is my bad - I didn't originally have a .gitignore file in the repo (if you've not come across a .gitignore file, it's just a file that lists the files that shouldn't be picked up by Git when any changes are made to it or files have been added to a specified directory). Files that are in the __pycache__ folder should be ignored as every time you run the application, they will change, which will fog up what needs to be reviewed in PRs and they don't impact the actual running of the application.

A .gitignore file was added to the repo a few days ago. If you remove the `__pycache__` folder, pull latest master, and run the application, these files won't show up in Git again :)